packages/commonwealth/client/scripts/views/pages/manage_community/chain_metadata_rows.tsx

```diff
@@ -55,6 +55,7 @@ export class ChainMetadataRows
   network: ChainNetwork;
   symbol: string;
   snapshot: string[];
+  snapshotString: string;
```

Comment:
why do we need this and the snapshot string array?

Reply:
I think this is because of a change made to the CWInput from when this was originally implemented- but tldr we need a variable to represent the state of the editor (i.e. "aave.eth, dydy.eth") and one that actually stores the snapshots as an array (i.e. ["aave.eth", "dydx.eth"]